R/bundle.R

```diff
@@ -263,7 +288,9 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters, a
   metadata <- list(appmode = appMode)
 
   # emit appropriate primary document information
-  primaryDoc <- ifelse(is.null(appPrimaryDoc), NA, appPrimaryDoc)
+  primaryDoc <- ifelse(is.null(appPrimaryDoc) ||
```

Comment:
Oh, that's confusing.. Until this point, `appPrimaryDoc` is either a doc or a shiny app, but here we make sure that it's a document. Does that mean it is mis-named?